basic/array_test1.c

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2017 National University of Singapore
+ */
+#include <klee/klee.h>
+#include <stdio.h>
+
+void test_array(char *x);
+
+char arr[5], answer;
+
+int main() {
+
+  klee_make_symbolic(arr, sizeof arr, "__arr8__arr");
+  klee_track_error(arr, "__arr8__arr_error");
```

Comment:
@domainexpert Perhaps the input arrays should also be prefixed with \__arr8__? For this example it is not important for the final result, as the input is not involved in the final error expression, but it might be good to document the use of it. (Sorry that my example test case did not contain this change)